.github/workflows/doc.yml

```diff
@@ -0,0 +1,26 @@
+name: Build LaTeX documentation
+on: [push]
+jobs:
+  build_latex:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up repository
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
```

Comment:
Not sure these lines are needed?

Reply:
They were in the example action for one of the stages, so left as is